tests/jerry/es2015/regression-test-issue-2669.js

```diff
@@ -0,0 +1,33 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
Where does the name of this test file come from? How is this related to PR 2669?

I see this snippet in the discussion of this PR, which is 2664.